Feature/Added doc for job cancellation (#177)
Avaiga/taipy-doc: docs/relnotes.md

**`taipy-core`**<br/>1.2.0

   - The data node of a scenario or a pipeline can now be accessed at the scenario or pipeline level.
   - When submitting a scenario, a pipeline or a task, a list of jobs or a Job will be returned.
   - A submitted job can now be cancelled and prevent from running. If the job has already been run, meaning a subprocess has already been dispatched to execute said job, the job can still be cancelled, but the subprocess will keep running until finished.

>    - When cancelling a job, the subsequent jobs of the cancelled job will be set to ABANDONED and will not be run.

Reviewer: In the release notes, I would put a single entry with fewer details. If they want more details, they can go to the user man.

```suggestion
   - A submitted job can be cancelled. The subsequent jobs are abandoned.
```

What do you think?

Author: Sounds good to me!
